src/elements/Toggle/Toggle.tsx

```diff
@@ -49,13 +49,13 @@ export class Toggle extends React.Component<ToggleProps> {
     return (
       <Flex width="100%" flexDirection="column">
         <Separator mb={2} />
-        <Header mt="-6px" onClick={this.toggleExpand} disabled={disabled}>
+        <Header mb={2} onClick={this.toggleExpand} disabled={disabled}>
           <Flex justifyContent="space-between" alignItems="center">
             <Sans
               size={this.props.textSize as SansSize}
               weight="medium"
               color="black100"
-              mt={0.3}
+              lineHeight="1.83"
```

Comment:
Since line-heights are typically defined [according to the font](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L160) it's better to use margins and padding and stick with the defaults from the theme.

Reply:
ah that makes sense. will tweak!